### ISSUE TITLE
Cannot (re)install addon

### ISSUE THREAD

**Issue by RalfJung** [rsjtdrjgfuzkfg/thunderbird-birthdaycalendar#16]
The addon worked great for me until recently, but when debugging a Thunderbird issue recently I disabled the extension -- and now I cannot get it to work again. I tried to reinstall to see if that helps, but after uninstalling I am now getting an error when I try to reinstall via the official addons website. It says the extension is corrupt.

In the error console I see lots of errors saying "Error: Internal error: Invalid string format origin".
```
Uncaught (in promise) Error: Internal error: Invalid string format origin
    parseSchema resource://gre/modules/Schemas.jsm:1659
    parseSchema resource://gre/modules/Schemas.jsm:3465
    parseSchema resource://gre/modules/Schemas.jsm:2304
    parseSchema resource://gre/modules/Schemas.jsm:3465
    parseProperty resource://gre/modules/Schemas.jsm:1825
    parseSchema resource://gre/modules/Schemas.jsm:1846
    parseSchema resource://gre/modules/Schemas.jsm:3465
    loadType resource://gre/modules/Schemas.jsm:3111
    initKey resource://gre/modules/Schemas.jsm:3101
    get resource://gre/modules/Schemas.jsm:3245
    ObjectType resource://gre/modules/Schemas.jsm:1907
    parseSchema resource://gre/modules/Schemas.jsm:1880
    parseSchema resource://gre/modules/Schemas.jsm:3465
    loadType resource://gre/modules/Schemas.jsm:3111
    initKey resource://gre/modules/Schemas.jsm:3101
    get resource://gre/modules/Schemas.jsm:3245
    normalize resource://gre/modules/Schemas.jsm:3557
    normalize resource://gre/modules/Schemas.jsm:3749
    _getNormalizedManifest resource://gre/modules/Extension.jsm:953
    parseManifest resource://gre/modules/Extension.jsm:993
```

I have no idea what is going on here. This is with TB 91.13.0.

**Comment by RalfJung** [comment]
This seems to be fixed with 102.2.2  :shrug:

**Comment by rsjtdrjgfuzkfg** [comment]
No idea what happened there, but on the first glance I believe this was an issue within Thunderbird and not with the add-on. Either way, I'm glad the issue resolved itself :)
